panda/src/display/graphicsStateGuardian.cxx

```diff
@@ -230,6 +230,7 @@ GraphicsStateGuardian(CoordinateSystem internal_coordinate_system,
   _supports_generate_mipmap = false;
   _supports_depth_texture = false;
   _supports_depth_stencil = false;
+  _supports_luminance_texture = false;
```

Comment:
Sure, I'm splitting that, as well as reviewing the comments regarding glGeomMunger_src.cxx, and the luminance support.
Probably I'll fire a new pull request, so you can close this one as rejected